README.md

```diff
@@ -37,3 +37,6 @@ dist\bin\bcp.exe algorithm asio assign bimap container date_time dynamic_bitset
 ```
 
 Replace the outdated files in 'external/lib' with those from 'boost_.../stage/lib' and those in 'external/include/boost' with  the ones in '_include/boost'.
+
+### Note for non-Visual Studio users
+Boost may also be build using this method in the regular Windows Command Prompt, though a different toolset needs to be specified (such as `--toolset=gcc`) and `.\b2` must be invoked as an executable (`b2.exe`).
```

Comment:
'be build' -> 'be built'
I know it's nitpicky, but as the Readme is te only file a user will read, we may as well be ;)